rpcs3/Emu/Io/PadHandler.cpp

```diff
@@ -199,11 +199,7 @@ std::tuple<u16, u16> PadHandlerBase::NormalizeStickDeadzone(s32 inX, s32 inY, u3
 // get clamped value between min and max
 s32 PadHandlerBase::Clamp(f32 input, s32 min, s32 max)
 {
-	if (input > max)
-		return max;
-	else if (input < min)
-		return min;
-	else return static_cast<s32>(input);
+	return std::clamp(static_cast<s32>(input), min, max);
```

Comment:
Stop, this is wrong because original comparison ``input > max`` is of float type. It should be std::clamp<float>(...).